api/src/api/app/controller/__init__.py

```diff
@@ -0,0 +1,3 @@
+from app.controller.groups_controller import GroupsController
+
+__all__ = ["GroupsController"]
```

Comment:
Shouldn't all other controllers be include here as well? Or why is just GroupsController mentioned here?